main.go

```diff
@@ -36,16 +36,16 @@ type link struct {
 
 // D3 is the structure used to construct the data for the frontend D3 graph.
 type D3 struct {
-	Nodes []node `json:"nodes"`
-	Links []link `json:"links"`
+	Nodes      []node `json:"nodes"`
+	Links      []link `json:"links"`
+	LastUpdate string `json:"lastUpdate"`
```

Comment:
> I kept LastUpdate field in the D3 data structures because it made more sense to do so (and it was far easier): logically it is the time when graph data was last updated, not rate limits data

Is it really necessary to move it?